data/themes/default/www/js/documentloader.js

```diff
@@ -16,7 +16,7 @@ $.loadScript = function (url, callback) {
     The website is now fully useable.
 */
 function loadSearchJs() {
-    $("#custom-scripts-search").load("search.js", function () {
+    $("#custom-scripts-search").load("js/search.js", function () {
```

Comment:
Not quite sure why we would need this `js/` prefix here? The other `*.js` files live in the same folder as `documentloader.js`